helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -93,8 +93,7 @@ internal class ListLicensesCommand : CliktCommand(
     private val ignoreExcludedRuleIds by option(
         "--ignore-excluded-rule-ids",
         help = "A comma separated list of rule names for which --omit-excluded should not have any effect."
-    ).convert { it.split(",").toList() }
-        .default(emptyList())
+    ).convert { File(it.expandTilde()) }.split(",").default(emptyList())
```

Comment:
This is still wrong, `.convert { File(it.expandTilde()) }` should go away as this no `File` option.

Reply:
Sorry.